doc/classes/AStar2D.xml

```diff
@@ -139,8 +139,10 @@
 			<return type="PackedInt64Array" />
 			<param index="0" name="from_id" type="int" />
 			<param index="1" name="to_id" type="int" />
+			<param index="2" name="return_partial_path" type="bool" default="false" />
 			<description>
 				Returns an array with the IDs of the points that form the path found by AStar2D between the given points. The array is ordered from the starting point to the ending point of the path.
+				Will return a partial path to a point that is disconnected if [param return_partial_path] is true, otherwise it will return an empty array. This value is false by default.
```

Comment:
Perhaps this parameter should be called `allow_partial_path`? I also think the description could be rephrased to be more clear. CC @AThousandShips